src/bp/core/app/botpress.ts

```diff
@@ -180,6 +181,20 @@ export class Botpress {
     startLocalActionServer({ appSecret: process.APP_SECRET, port })
   }
 
+  private async maybeStartLocalSTAN() {
+    const config = await this.moduleLoader.configReader.getGlobal('nlu')
+
+    if (config.STANUrl) {
+      return
+    }
+    startLocalSTANServer({
+      languageURL: config.languageSources[0].endpoint,
```

Comment:
Are we sure there's at least one ? Also, let's make sure we don't forget to add multiple languageSources with auth tokens in config